compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/schema/generator/GeneratorUtils.java

```diff
@@ -34,6 +36,9 @@ public class GeneratorUtils {
     private GeneratorUtils() {
     }
 
+    private static final String UNICODE_REGEX = "\\\\(\\\\*)u\\{([a-fA-F0-9]+)\\}";
+    public static final Pattern UNICODE_PATTERN = Pattern.compile(UNICODE_REGEX);
```

Comment:
Shall we mark this as `private` cos it is not used anywhere else ?
```suggestion
    private static final Pattern UNICODE_PATTERN = Pattern.compile(UNICODE_REGEX);
```